node/node.go

```diff
@@ -434,6 +439,7 @@ func (n *Node) GetESDTData(address, tokenID string, nonce uint64, options api.Ac
 	return esdtToken, blockInfo, nil
 }
 
+// getTokensIDsWithFilter returns token IDs based on filters, works only on metachain
```

Comment:
Can delete this, we only add comments for exported functions. If the func name doesn't add any value to the func description, it can be removed